cluster/stone-prd-rh01/tenants/rhtap-build-tenant/release_plan.yaml

```diff
@@ -33,3 +33,14 @@ metadata:
 spec:
   application: build-trusted-artifacts
   target: rh-managed-rhtap-ser-tenant
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+  name: rh-syft-registry-redhat-io
+  namespace: rhtap-build-tenant
+  labels:
+    release.appstudio.openshift.io/auto-release: 'false'
```

Comment:
Why false?

Reply:
We want to manually release meaningful changes, we don't want every build to be released.

I'm hoping that's what this label does, couldn't find the documentation for it :sweat_smile:

Reply:
I'll let @scoheb or @gbenhaim correct me on this one. But I understand this label is for the release pipeline to be run automatically after every merge to main. That will create and image and push it to the repo you set up in the ReleasePlanAdmission. For deploying itself, that's on app interface, so I don't know if you prefer to manually create releases here or configure app interface to get the ones you want from the quay repo

Reply:
RH-syft is configured to release to registry.redhat.io. Pushing to registry.redhat.io on every merge to main is what we want to avoid